### PR TITLE
Update transformer.md

### PR DESCRIPTION
**Pull request by heiligerl**
- Fixed minor inconsistency in the example between the mxnet and the PyTorch version. (Alternatively to removing the *eval* statements, the evaluation mode could remain and the argument *track_running_stats* in *nn.BatchNorm1d* could be set to *False*)
- Added a tab for the PyTorch version in "Add and Norm".

By submitting this pull request, I confirm that you can use, modify,
copy, and redistribute this contribution, under the terms of your
choice.

### CODE REVIEW

**Comment by mli** [comment]
Job d2l-en/PR-1463/1 is complete. 
Check the results at http://preview.d2l.ai/d2l-en/PR-1463/

**Comment by mli** [comment]
Job d2l-en/PR-1463/2 is complete. 
Check the results at http://preview.d2l.ai/d2l-en/PR-1463/

**Review by AnirudhDagar — APPROVED** [review]
Thanks @heiligerl LGTM!
